yb-voyager/cmd/importData.go

```diff
@@ -855,39 +858,49 @@ func doOneImport(task *SplitFileImportTask, connPool *tgtdb.ConnectionPool) {
 					}
 				}
 				res, err := conn.PgConn().CopyFrom(context.Background(), reader, copyCommand)
-				rowsCount = res.RowsAffected()
+				rowsAffected = res.RowsAffected()
 
+				if err != nil && (strings.Contains(err.Error(), "invalid input syntax") ||
+					strings.Contains(err.Error(), "violates unique constraint")) {
```

Comment:
Add a comment here stating the "violates unique constraint" error will be handled outside of `WithConn()`.

Reply:
What about `Sending too long RPC message`? We should not retry for that error as well.

Reply:
adding that